### DISCUSSION TITLE
Can't longer access swizzin panel via webbrowser

### DISCUSSION THREAD

**Discussion by Lorenz1508** [swizzin/swizzin#728] [Q&A + Help] [answered]
Hello

after i accidentally  wiped completely all my passwords in the password manager. I can no longer access my swizzin panel. is there a possibility to change the password by console/terminal?

**Comment by flying-sausages** [comment, marked as answer]
https://swizzin.ltd/getting-started/box-basics

`box chpasswd <your-username>`
